servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -1,21 +1,13 @@
-import connexion
-
+from jobs.controllers.utils import generator, time_convert, providers
+from jobs.models.aggregation import Aggregation
+from jobs.models.aggregation_entry import AggregationEntry
 from jobs.models.aggregation_response import AggregationResponse
 from jobs.models.status_count import StatusCount
 from jobs.models.status_counts import StatusCounts
-from jobs.models.aggregation import Aggregation
-from jobs.models.aggregation_entry import AggregationEntry
-
-from datetime import date, datetime
-from typing import List, Dict
-from six import iteritems
-from ..util import deserialize_date, deserialize_datetime
-from werkzeug.exceptions import NotImplemented
 
 
 def get_job_aggregations(timeFrame, projectId=None):
-    """
-    Query for aggregated jobs in the given time frame.
+    """Query for aggregated jobs in the given time frame.
```

Comment:
Have you looked at the jobs_controller test? I would recommend adding tests in this PR (or else, in a follow-up PR, but before you complete the full API implementation).

Reply:
OK, will do it in the next PR.